lib/potentialbuild.rb

```diff
@@ -776,10 +774,8 @@ def post_results(compiler, pending)
                        (test_results_passed.to_f / test_results_total.to_f) * 100.0
                      end
 
-      if test_percent > @config.test_pass_limit
+      if (test_percent - 100.0).abs < Float::EPSILON
         test_color = 'green'
-      elsif test_percent > @config.test_warn_limit
-        test_color = 'yellow'
```

Comment:
Before we would use the test pass/warn limits to decide what to report.  Now we just check if it is equal (floating point) to 100% and if it's not, fail!  I guess technically this would reveal a failure if it goes over 100%.... but WHAT?